assets/scripts/modules/moj.skip-to-content.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+  'use strict';
+
+  moj.Modules.SkipToContent = {
+    init: function() {
+      this.cacheEls();
+      this.bindEvents();
+    },
+
+    bindEvents: function() {
+      this.$body
+        .on('click', '.skip-link', this._focusOnContent)
+        .on('blur', '#content', this._removeFocus);
```

Comment:
Can we make this less specific? So the ID doesn't have to be "content". 

Maybe just bind the blur event once in the focusOnContent method?